code/hello-world-tutorial/hello.tape

```diff
@@ -0,0 +1,11 @@
+Output hello.gif
+Set Theme { "name": "Aardvark Blue", "black": "#191919", "red": "#aa342e", "green": "#4b8c0f", "yellow": "#dbba00", "blue": "#1370d3", "purple": "#c43ac3", "cyan": "#008eb0", "white": "#bebebe", "brightBlack": "#454545", "brightRed": "#f05b50", "brightGreen": "#95dc55", "brightYellow": "#ffe763", "brightBlue": "#60a4ec", "brightPurple": "#e26be2", "brightCyan": "#60b6cb", "brightWhite": "#f7f7f7", "background": "#102040", "foreground": "#dddddd", "cursor": "#007acc", "selection": "#bfdbfe" }
+Set Width 800
+Set Height 600
```

Comment:
What do you think about making the height a little smaller? It might make it easier to read on laptop screens.

Reply:
I think this is probably about the right size for laptops. Looks good on a phone too. If we go much smaller than this, then it gets a bit cramped to show the cargo output. Learning towards keeping it unless you strongly think it would be better.

Reply:
Maybe with font size 18? 

Here's the alternate screen demo compared:

![image](https://github.com/ratatui-org/ratatui-book/assets/1813121/c841af82-0aa2-4d17-8c58-4a5d666ce8fe)

This is on my desktop with more vertical room. On my laptop when scrolling through the page, the image would take up the entire vertical viewport with no text above and below.

Reply:
![hello](https://github.com/ratatui-org/ratatui-book/assets/381361/98eee556-6283-4aa5-a068-99392e1a5dda)
![hello](https://github.com/ratatui-org/ratatui-book/assets/381361/b324807e-915f-45b3-a4a2-d3b419eec387)